libs/net/models/Settings/ReportContentSettingsModel.cs

```diff
@@ -25,6 +26,7 @@ public ReportContentSettingsModel(Dictionary<string, object> settings, JsonSeria
         this.ExcludeReports = settings.GetDictionaryJsonValue("excludeReports", Array.Empty<int>(), options)!;
         this.ShowLinkToStory = settings.GetDictionaryJsonValue("showLinkToStory", false, options)!;
         this.HighlightKeywords = settings.GetDictionaryJsonValue("highlightKeywords", false, options)!;
+        this.IncludeStory = settings.GetDictionaryJsonValue("includeStory", false, options)!;
```

Comment:
What is this option for?

Reply:
Same as first comment above.

Reply:
Each section has a setting `showFullStory` which sounds like it is related.  I don't see why we need this property here.